index.bs

```diff
@@ -822,6 +818,20 @@ When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a ref
 
 </div>
 
+<div class="algorithm" data-algorithm="get-offset-space">
+The <dfn method for="XRReferenceSpace">getOffsetReferenceSpace(|originOffset|)</dfn> method MUST perform the following steps when invoked:
+
+  1. Let |base| be the {{XRReferenceSpace}} the method was called on.
+  1. If |base| is an instance of {{XRStationaryReferenceSpace}}, let |offsetSpace| be a new {{XRStationaryReferenceSpace}}.
+  1. Else if |base| is an instance of {{XRBoundedReferenceSpace}}, let |offsetSpace| be a new {{XRBoundedReferenceSpace}} and set |offsetSpace|'s {{boundsGeometry}} to |base|'s {{boundsGeometry}}, with each point transformed by the {{XRRigidTransform/inverse}} of |originOffset|.
+  1. Else if |base| is an instance of {{XRUnboundedReferenceSpace}}, let |offsetSpace| be a new {{XRUnboundedReferenceSpace}}.
+  1. Else let |offsetSpace| be a new {{XRReferenceSpace}}.
+  1. Set |offsetSpace|'s [=native origin=] to |base|'s [=native origin=].
+  1. Set |offsetSpace|'s [=origin offset=] to the result of [=multiply transforms|multiplying=] |originOffset| by |base|'s [=origin offset=].
```

Comment:
Someone not me should check that this math is in the right order?

Reply:
It's correct, here `new * old` will provide a transformation where `old` is applied first and `new` second.